integration/mediation-tests/tests-mediator-2/src/test/java/org/wso2/carbon/esb/message/processor/test/MessageProcessorAdminTestCase.java

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.esb.message.processor.test;
+
+import org.apache.axiom.om.OMElement;
+import org.apache.axiom.om.util.AXIOMUtil;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import org.wso2.carbon.automation.engine.context.AutomationContext;
+import org.wso2.carbon.automation.engine.context.TestUserMode;
+import org.wso2.carbon.integration.common.utils.LoginLogoutClient;
+import org.wso2.carbon.message.store.stub.MessageInfo;
+import org.wso2.esb.integration.common.clients.mediation.MessageProcessorClient;
+import org.wso2.esb.integration.common.clients.mediation.MessageStoreAdminClient;
+import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
+import org.wso2.esb.integration.common.utils.servers.axis2.SampleAxis2Server;
+
+import javax.xml.namespace.QName;
+import java.util.Iterator;
+
+/**
+ * Tests to verify service behaviour of both Message Processor Admin Service and Message Store Admin Service.
+ */
+public class MessageProcessorAdminTestCase extends ESBIntegrationTest {
+
+    private MessageProcessorClient messageProcessorClient;
+    private MessageStoreAdminClient messageStoreAdminClient;
+
+    private AutomationContext esbContext;
+    private String sessionCookie;
+    private LoginLogoutClient loginLogoutClient;
+
+    private final String PROCESSOR_NAME = "mspAdminTestInMemoryMessageProcessor";
+    private final String PROXY_NAME = "mspAdminTestProxy";
+    private final String STORE_NAME = "mspAdminTestInMemoryMessageStore";
+
+    @BeforeClass(alwaysRun = true, description = "Test Car with Mediator deployment")
+    protected void setup() throws Exception {
+        super.init();
+
+        esbContext = new AutomationContext("ESB", TestUserMode.SUPER_TENANT_ADMIN);
+        loginLogoutClient = new LoginLogoutClient(esbContext);
+        sessionCookie = loginLogoutClient.login();
+
```

Comment:
do not need bellow calls. super.init() already make a authentication

Reply:
Agreed. Fixed with latest commit.